facade/src/main/java/com/redhat/lightblue/migrator/facade/TimeoutConfiguration.java

```diff
@@ -118,7 +118,17 @@ public long getMS(String methodName, FacadeOperation op, Type type) {
                 log.debug("{} config not found for bean {} using global timeout", type, beanName);
             }
 
-            timeout = defaultTimeoutMS;
+            switch (type) {
+                case timeout: {
+                    timeout = defaultTimeoutMS; break;
+                }
+                case slowwarning: {
+                    timeout = 2 * defaultTimeoutMS; break;
```

Comment:
Yes, it's used only for logging. I'm not sure what you mean by 
> let the constructor handle this?

Different types of timeouts may have different values. There is only one TimeoutConfiguration instance per facade.